src/docs/error-handling.dox

```diff
@@ -31,15 +31,15 @@ exclusive access when the objects might be in active use by other threads.
 
 @section error_sub_level Sub-level errors
 
-From the addition of WT_SESSION::get_last_error function and sub-level error codes, users are now available to
-call the API in order to get the last error from the previous session API call attached with more contextual information.
-These contextual information come in the form of sub-level error codes and error messages, which aim to provide
-users the ability to programmatically decide what action to take based on information. The  sub-level error code and error message 
-are optional, and default to WT_NONE and an empty string respectively. The error message string is 
-owned by the session, and is invalidated after each session API call.
-
-If the last API call was successful, WT_SESSION::get_last_error will give the error code as 0,
-sub-level error code as WT_NONE, and error message as "last API call was successful".
+From the addition of WT_SESSION::get_last_error function and sub-level error codes, users
+are available to get the provide more verbose information about the last
```

Comment:
This sentence seems confusing. Did you mean
```suggestion
can get more detailed information about the last
```

Reply:
@etienneptl 's suggestion is much clearer.

Reply:
Thanks fixed up [5058d85](https://github.com/wiredtiger/wiredtiger/pull/11427/commits/5058d85dc0806548b7d763abfb402c0bc84f4be2)